example.env

```diff
@@ -16,3 +16,5 @@ CLOUDINARY_API_SECRET=
 TEST_CLOUDINARY_CLOUD_NAME=
 TEST_CLOUDINARY_API_KEY=
 TEST_CLOUDINARY_API_SECRET=
+
+SENTRY_DSN=
```

Comment:
GitHub likes us to leave one empty line at the bottom of files that we add to commits.